etna/articles/models.py

```diff
@@ -226,3 +229,166 @@ def latest_items(self) -> Tuple["ArticlePage"]:
 
     parent_page_types = ["articles.ArticleIndexPage"]
     subpage_types = []
+
+
+class RecordArticlePage(
+    TopicalPageMixin, ContentWarningMixin, TeaserImageMixin, MetadataPageMixin, BasePage
+):
+    parent_page_types = ["collections.ExplorerIndexPage"]
+    subpage_types = []
+
+    standfirst = models.CharField(
+        verbose_name=_("standfirst"), max_length=350, blank=False
```

Comment:
Note: I think using the `verbose_name` here is redundant, my understanding that there will be no difference than just allowing the field name to be rendered?

Reply:
Yes, this is on purpose. Whatever the project, I try to mark whatever simple strings I can for translation using `django.utils.translation.gettext_lazy`, as it takes very little effort to do as you go along.